### PR TITLE
[VL] use velox config lib to extract the configurations from spark

### PR DESCRIPTION
**Pull request by FelixYBW**
The PR is to avoid some other guy spending hours on stupid mistake by using 

`conf.set('spark.hadoop.fs.s3a.use.instance.credentials',True)`
instead of 
`conf.set('spark.hadoop.fs.s3a.use.instance.credentials','true')`

The root cause is that we use got=='true' to check if the flag is set.

The PR use velox's config lib to parse the configurations from Spark.

True can be set by 'true', 'True', 1, 'ON', 'Y', 'T', etc.

limitation: 'KB', 'MB', 'GB' is still not supported, only number is expected

### CODE REVIEW

File: cpp/velox/compute/VeloxBackend.cc [hunk 1]
```diff
@@ -120,79 +120,49 @@ void VeloxBackend::printConf(const std::unordered_map<std::string, std::string>&
...
   // Init glog and log level.
-  {
-    auto vlogLevel = stoi(getConfigValue(conf, kGlogVerboseLevel, kGlogVerboseLevelDefault));
-    auto severityLogLevel = stoi(getConfigValue(conf, kGlogSeverityLevel, kGlogSeverityLevelDefault));
-    FLAGS_v = vlogLevel;
-    FLAGS_minloglevel = severityLogLevel;
-    FLAGS_logtostderr = true;
-    google::InitGoogleLogging("gluten");
-  }
+  auto veloxmmecfg = std::make_shared<facebook::velox::core::MemConfigMutable>(conf);
```
Comment (zhztheplayer) [inline cpp/velox/compute/VeloxBackend.cc:123]: Looks like a typo? s/veloxmmecfg/veloxmemcfg/ ?

**Review by zhztheplayer — COMMENTED** [review]
Thanks!

File: cpp/velox/compute/VeloxBackend.cc (hunk 1 above)
Comment (FelixYBW) [reply cpp/velox/compute/VeloxBackend.cc:123]: Thank you, good catch

File: cpp/velox/compute/VeloxBackend.cc [hunk 2]
```diff
@@ -208,50 +178,35 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
   }
 
   std::unordered_map<std::string, std::string> s3Config({});
```
Comment (yma11) [inline cpp/velox/compute/VeloxBackend.cc:180]: remove variable `s3Config` as it's not used anymore.

**Review by zhztheplayer — APPROVED** [review]

File: cpp/velox/compute/VeloxBackend.cc (hunk 2 above)
Comment (FelixYBW) [reply cpp/velox/compute/VeloxBackend.cc:180]: Oh, missed your comments. Let's remove it in next PR

**Pull request merged**